### PR TITLE
mavlink: move streams to streams headers

### PR DESCRIPTION
**Pull request by antonio-sc66**
Move some Mavlink Streams from mavlink_messages.cpp to streams/

Changes have been tested locally by runing _make px4_sitl_test_ and with _px4_sitl_default jmavsim_ and checking with _QGroundControl_

### CODE REVIEW

File: src/modules/mavlink/streams/OPTICAL_FLOW.hpp [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+#ifndef OPTICAL_FLOW_HPP
```
Comment (dagar) [inline src/modules/mavlink/streams/OPTICAL_FLOW.hpp:34]: For consistency can you keep this identical to the Mavlink message name?

```suggestion
#ifndef OPTICAL_FLOW_RAD_HPP
```

File: src/modules/mavlink/mavlink_messages.cpp [hunk 2]
```diff
@@ -4321,11 +3939,15 @@ static const StreamListItem streams_list[] = {
 	create_stream_list_item<MavlinkStreamHILActuatorControls>(),
 	create_stream_list_item<MavlinkStreamPositionTargetGlobalInt>(),
 	create_stream_list_item<MavlinkStreamLocalPositionSetpoint>(),
+#if defined(ATTITUDE_TARGET_HPP)
 	create_stream_list_item<MavlinkStreamAttitudeTarget>(),
+#endif // ATTITUDE_TARGET_HPP
 	create_stream_list_item<MavlinkStreamRCChannels>(),
 	create_stream_list_item<MavlinkStreamManualControl>(),
 	create_stream_list_item<MavlinkStreamTrajectoryRepresentationWaypoints>(),
+#if defined(OPTICAL_FLOW_HPP)
```
Comment (dagar) [inline src/modules/mavlink/mavlink_messages.cpp:3948]:
```suggestion
#if defined(OPTICAL_FLOW_RAD_HPP)
```

**Comment by dagar** [comment]
One minor nitpick about naming consistency, but otherwise looks good.

**Comment by antonio-sc66** [comment]
Should I also change the OPTICAL_FLOW.hpp file name to OPTICAL_FLOW_RAD.hpp?. Looks strange to have OPTICAL_FLOW_RAD_HPP definition with a different filename.

**Comment by dagar** [comment]
> Should I also change the OPTICAL_FLOW.hpp file name to OPTICAL_FLOW_RAD.hpp?

Yes that too.

**Review by dagar — APPROVED** [review]

**Pull request merged**